<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtUsername">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>180</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Giri≈ü</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
